main.js

```diff
@@ -34,6 +34,7 @@ const srcDoc = `
  * and options. It waits for everything to initialize and returns the fully initialized
  * editor element, which has access to a controller and other methods (see docs)
  * @param {RenderOpts} options
+ * @param {Object.<string, string>} Record of CSS Variables and their values to be applied to the editor
```

Comment:
the conventional type for this is `Record<string, string>`

Reply:
Right, you can use typescript in those types as well fyi https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html